chainstate/test-suite/src/tests/output_timelock.rs

```diff
@@ -61,7 +61,7 @@ fn output_lock_until_height(#[case] seed: Seed) {
         );
 
         // attempt to create the next block, and attempt to spend the locked output
-        assert_eq!(
+        assert!(matches!(
```

Comment:
In all these tests, you do have the OutPoint, right? Why not put it in the assert instead of using matches? Please understand that using assert + matches makes debugging much, much harder.